awesomplete.js

```diff
@@ -169,14 +169,15 @@ _.prototype = {
 
 	next: function () {
 		var count = this.ul.children.length;
-
-		this.goto(this.index < count - 1? this.index + 1 : -1);
+		var defaultPos = count ? 0 : -1;
+		this.goto(this.index < count - 1? this.index + 1 : defaultPos);
```

Comment:
I can :) 
_imho_ I don't think it's a good idea, because a ternary inside another one is not easy to read.